### PR TITLE
Buy sell improvements

### PR DESCRIPTION
**Pull request by sixtedemaupeou**
## Description
882 914

## Change Type
- Bug Fix

## Testing Steps
Detail the steps required for the reviewer(s) to verify and test these changes.

## PR Creator Checklist
- [x] Code compiles correctly
- [ ] No lint issues exist (verified via `yarn lint`)
- [x] All unit tests pass (verified via `yarn test`)
- [x] Updated `README.md` and other documentation (if necessary)

## PR Reviewer Checklist
- [ ] Change validated and application spot checked
- [ ] Code styles and best practices met
- [ ] Code is readable and commented when necessary

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/components/Form/TabMenuBuySellStatus/index.js [hunk 1]
```diff
@@ -9,6 +13,7 @@ class TabMenuBuySellStatusContainer extends React.PureComponent {
   }
 
   handleClick (value) {
+    this.props.coinifyActions.coinifyNextCheckoutStep('checkout')
```
Comment (prwelber) [inline packages/blockchain-wallet-v4-frontend/src/components/Form/TabMenuBuySellStatus/index.js]: This generic Tab menu shouldn't know anything about Coinify

File: packages/blockchain-wallet-v4-frontend/src/data/preferences/reducers.js [hunk 2]
```diff
@@ -46,6 +47,9 @@ const preferences = (state = INITIAL_STATE, action) => {
       const { displayed } = payload
       return assoc('showBitcoinCashWelcome', displayed, state)
     }
+    case AT.HIDE_KYC_COMPLETED: {
+      return assoc('showKycCompleted', false, state)
```
Comment (prwelber) [inline packages/blockchain-wallet-v4-frontend/src/data/preferences/reducers.js]: I think we want to hide the KYC notification permanently once the user successfully completes it. Does this do that?

Comment (sixtedemaupeou) [reply packages/blockchain-wallet-v4-frontend/src/data/preferences/reducers.js]: Yes, as soon as the action is called showKycCompleted becomes false and then this is stored along with all other preferences in localStorage via reduxPersistStore

Comment (prwelber) [reply packages/blockchain-wallet-v4-frontend/src/data/preferences/reducers.js]: Nice!

**Pull request merged**